src/cms/templates/linkcheck/links_by_filter.html

```diff
@@ -0,0 +1,59 @@
+{% extends '_base.html' %}
+{% load i18n %}
+
+{% block content %}
+  <h1 class="heading">
+    {% if link_filter == 'valid' %}
+      {% trans 'Valid links' %} ({{ number_valid }})
+    {% elif link_filter == 'invalid' %}
+      {% trans 'Invalid links' %} ({{ number_invalid }})
+    {% elif link_filter == 'unchecked' %}
+      {% trans 'Unchecked links' %} ({{ number_unchecked }})
+    {% else %}
+      {% trans 'Ignored links' %} ({{ number_ignored }})
```

Comment:
```suggestion
      {% trans 'Valid links' %}
    {% elif link_filter == 'invalid' %}
      {% trans 'Invalid links' %}
    {% elif link_filter == 'unchecked' %}
      {% trans 'Unchecked links' %}
    {% else %}
      {% trans 'Ignored links' %}
```
Since the counters are already included in the line below, I'd omit them in the headings.